cli/pcluster/utils.py

```diff
@@ -407,8 +407,7 @@ def get_availability_zone_of_subnet(subnet_id):
                     subnet_id, e.response.get("Error").get("Message")
                 )
             )
-            raise
-    return cache[subnet_id]
+    return cache.get(subnet_id)
```

Comment:
This bug came from [my commit](https://github.com/aws/aws-parallelcluster/commit/ae178aa4f9a0889f5b52e767ffee8d23cbcac5d1). Before my commit, the code here was `return cache[subnet_id]` without `raise`. The new change fixes the bug. Thank you!